lib/nrf_modem_lib/nrf_modem_at_cmd_filter.c

```diff
@@ -0,0 +1,305 @@
+/*
```

Comment:
The commit message dose not mention `nrf_modem_at_cmd_filter` introduction at all (and there is no entry in the changelog). Anyway, it's seems pretty separate from manifest update, would it make sense to make a separate commit out of it?